crates/node_binding/src/compilation/mod.rs

```diff
@@ -736,6 +736,7 @@ impl JsCompilation {
         let dependency = if let Some(map) = include_dependencies_map.get(&js_dependency.request)
           && let Some(dependency) = map.get(&options)
         {
+          js_dependency.dependency_id = Some(*dependency.id());
```

Comment:
what will this affect?

Reply:
cause bug when pass to method for module graph

Reply:
you can see the test case